Sources/App/Core/AppEnvironment.swift

```diff
@@ -179,7 +189,7 @@ extension AppEnvironment {
         },
         mastodonPost: Mastodon.post(client:message:),
         metricsPushGatewayUrl: { Environment.get("METRICS_PUSHGATEWAY_URL") },
-        plausibleAPIReportingSiteID: { Environment.get("PLAUSIBLE_API_REPORTING_SITE_ID") },
+        plausibleBackendReportingSiteID: { Environment.get("PLAUSIBLE_BACKEND_REPORTING_SITE_ID") },
```

Comment:
I've added the corresponding new variables to `dev` and `prod` and will remove the old one after deployment.